src/Meilisearch/Settings.cs

```diff
@@ -55,5 +55,11 @@ public class Settings
         /// </summary>
         [JsonPropertyName("sortableAttributes")]
         public IEnumerable<string> SortableAttributes { get; set; }
+
+        /// <summary>
+        /// Gets or sets the typo tolerance configuration attributes.
```

Comment:
```suggestion
        /// Gets or sets the typo tolerance attributes.
```
Not sure about that but it seems to be more understandable without configuration for me.